task_6_9.py

```diff
@@ -0,0 +1,22 @@
+import string
+
+
+def count_letters(content, letter_type):
```

Comment:
nice function but you don't need it anymore, as you are going to calculate only lowercase letters